docs/java-rest/high-level/cluster/put_settings.asciidoc

```diff
@@ -0,0 +1,129 @@
+[[java-rest-high-cluster-put-settings]]
+=== Cluster Put Settings API
```

Comment:
I am sorry, can you change the title to Cluster Update Settings API, like we have in the es docs? Confusing, I know :)